mysql scripts/table_creation/appointmentDB.sql

```diff
@@ -0,0 +1,74 @@
+-- VITA Project
+-- UNL CSE Ambassadors 2017
+
+USE vita;
+
+DROP TABLE IF EXISTS AppointmentQuestionAnswer;
+DROP TABLE IF EXISTS Appointment;
+DROP TABLE IF EXISTS Location;
+DROP TABLE IF EXISTS UserAnswer;
+DROP TABLE IF EXISTS PossibleAnswer;
+DROP TABLE IF EXISTS Question;
+DROP TABLE IF EXISTS QuestionAccessories;
+
+CREATE TABLE QuestionAccessories (
```

Comment:
Additionally, this table could include html attributes like spellcheck, required, readonly, disabled, etc. It may seem excessive but if we're going this far to engineer this form, we might want to consider adding more of these constraints.

The alternative I see is to outsource all of the validation and constraint to @CollinsSpencer jQuery validation voodoo.